public/kcl-samples/pipe-with-bend/main.kcl

```diff
@@ -14,22 +15,14 @@ bendAngle = 90
 sketch000 = startSketchOn("XZ")
 
 // create a profile for the outer diameter
-outerProfile = circle(
-  sketch000,
-  center = [bendRadius, 0],
-  radius = outerDiameter / 2
-)
+outerProfile = circle(sketch000, center = [bendRadius, 0], radius = outerDiameter / 2)
 
 // create a profile for the inner diameter
-innerProfile = circle(
-  sketch000,
-  center = [bendRadius, 0],
-  radius = innerDiameter / 2
-)
+innerProfile = circle(sketch000, center = [bendRadius, 0], radius = innerDiameter / 2)
 
 // create the profile of the pipe
 pipeProfile = outerProfile
   |> hole(innerProfile, %)
 
 // revolve the pipe profile at the desired angle
-pipe = revolve(pipeProfile, axis = "Y", angle = bendAngle)
+pipe = revolve({ axis = 'Y', angle = bendAngle }, pipeProfile)
```

Comment:
`revolve()` uses keyword args now https://github.com/KittyCAD/modeling-app/pull/5873.

Reply:
ahh, thank you